### PR TITLE
Cards3

### PR DESCRIPTION
**Pull request by CotyJ**
Add Prompt Card component and styling to JudgeView and PlayerView.

Changed .text for static cards to .body for live/custom cards.

### CODE REVIEW

File: src/card/PCard.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Card.module.css';
+
+export default class Card extends React.Component {
+
+  drag = (e) => {
```
Comment (codingavatar) [inline src/card/PCard.js]: I don't think prompt cards need to be draggable

File: src/card/Card.module.css [hunk 2]
```diff
@@ -44,15 +44,15 @@
   font-size: 1.2rem;
   font-weight: 600;
   height: 40px;
-  left: 28%;
+  left: 8%;
   margin: 5px 5px 0px 5px;
   padding: 4px;
   position: absolute;
   width: 40%;
 }
```
Comment (codingavatar) [inline src/card/Card.module.css:53]: Please make answer card's text scrollable

**Review by codingavatar — APPROVED** [review]
Looks good!

**Review by rochun — APPROVED** [review]
Looks good! I second what Jessica said about the overflow-y property for card overflows.

**Pull request merged**